llvm/include/llvm/IR/IntrinsicsSPIRV.td

```diff
@@ -82,5 +82,6 @@ let TargetPrefix = "spv" in {
     [llvm_anyint_ty, LLVMScalarOrSameVectorWidth<0, LLVMVectorElementType<0>>],
     [IntrNoMem, Commutative] >;
   def int_spv_wave_is_first_lane : DefaultAttrsIntrinsic<[llvm_i1_ty], [], [IntrConvergent]>;
+  def int_spv_wave_read_lane_at : DefaultAttrsIntrinsic<[llvm_any_ty], [LLVMMatchType<0>, llvm_i32_ty], [IntrConvergent]>;
```

Comment:
`wave_read_lane_at` would be better as one word.

Probably should update `wave_is_first_lane` in a follow cleanup on pr.

Reply:
Added a commit to change the name to `waveReadLaneAt` as described in below comment.